### DISCUSSION TITLE
which cpu do you recommend to use combined with a Google Coral?

### DISCUSSION THREAD

**Discussion by MaXeN9818** [blakeblackshear/frigate#4982] [Ask A Question] [answered]
I purchased "_pre-ordered, hope it gets to me within the 180 day guarantee_". But now I have a doubt. To use two cameras at 1080 which cpu do you recommend?
I have a mini pc with an Intel Celeron N3050 (2 cores at 2.16 GHz) or do you recommend me to buy something better?

**Comment by NickM-27** [comment, marked as answer]
> I have a mini pc with an Intel Celeron N3050 (2 cores at 2.16 GHz) or do you recommend me to buy something better?

I've seen some users have issues with this one as the iGPU decoder does not work very well, https://docs.frigate.video/frigate/hardware shows what's recommended

**Reply by MaXeN9818** [reply]
You convinced me, I bought the
MINISFORUM GK41 since exactly a few minutes ago it was put on a strong discount "admit it you work for amazon".
Thanks for the quick response
